Makefile

```diff
@@ -14,4 +16,9 @@ integration_w_race:
 	go test -race -testserver -allocator -v ./...
 	go test -race -integration -allocator -testserver -v ./...
 
+COUNT ?= 1
+BENCHMARK_PATTERN ?= "."
 
+benchmark:
+	go test -integration -run=NONE -bench=$(BENCHMARK_PATTERN) -benchmem -memprofile memprofile.out -count=$(COUNT)
+	go tool pprof -svg -output=memprofile.svg memprofile.out
```

Comment:
Isn't -memprofile expensive? I always benchmark without profiling if I can avoid it.

Reply:
For the full sweet of benchmarks, it was 4m 43.640s with memprofile, and 4m 42.839s without memprofile.

Less than a second on a nearly 5 minute run time isn’t so bad?

Reply:
I’ll switch the `Makefile` to have both options though, same as the above `_w_race` but `_w_memprofile`.